environments/development.env

```diff
@@ -6,6 +6,7 @@ NODE_ENV=development
 UBUNTU_SSO_URL=https://login.ubuntu.com
 OPENID_VERIFY_URL=http://localhost:8000/login/verify
 LP_API_URL=https://api.launchpad.net
+STORE_MACAROON_API_URL=https://myapps.developer.ubuntu.com/dev/api
```

Comment:
As we are talking to production LP only, I guess it's fine to talk to production store to check name registration.